source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -920,6 +920,15 @@ a manner idiomatic to the Driver and Language.
       address: <pool address>;
     }
 
+    /**
+     *  Thrown when the driver attempts to check out a
+     *  Connection from a paused Connection Pool
+     */
+    interface PoolClearedError extends RetryableError {
+      message: 'Attempted to check out a Connection from paused connection pool';
```

Comment:
I like it. Even better if the driver can include the reason why the pool was cleared to begin with. Updated to:
```
    interface PoolClearedError extends RetryableError {
      message: 'Connection pool for <pool address> was cleared because another operation failed with: <original error which cleared the pool>';
```